src/host/polyfills/simpleView.ts

```diff
@@ -121,15 +125,58 @@ export function applySetTabIconPatch(content: string) {
 export function applyAppendTabPatch(content: string) {
     // The appendTab function chooses which tabs to put in the tabbed pane header section
     // showTabElement and selectTab are only called by tabs that have already been appended via appendTab.
-    const allowedTabs = [
+    const elementsTabs = [
         "elements",
         "Styles",
         "Computed",
         "accessibility.view",
         "elements.domProperties",
         "elements.domBreakpoints",
         "elements.eventListeners",
-        "network",
+    ];
+
+    const condition = elementsTabs.map((tab) => {
+        return `id !== '${tab}'`;
+    }).join(" && ");
+
+    const appendTabWrapper =
+        /appendTab\(id,\s*tabTitle\s*,\s*view,\s*tabTooltip,\s*userGesture,\s*isCloseable,\s*index\)\s*{/;
+    const injectionPoint = /}(\s|\n)*export\s*const\s*Events={/;
+
+    // Injecting our verifications by redirecting appendTab to appendTabOverride
+    if (content.match(appendTabWrapper)) {
+        content = content.replace(
+            appendTabWrapper,
+            `appendTabOverride(id, tabTitle, view, tabTooltip, userGesture, isCloseable, index) {`);
+    } else {
+        return null;
+    }
+
+    // We then replace with the verifications itself.
+    if (content.match(injectionPoint)) {
+        content = content.replace(
+            injectionPoint,
+            `${getApprovedTabs.toString().slice(9)};

```

Comment:
NIT: I see now that here is where we are injecting the getApprovedTabs function.  Might be worth leaving a comment here explaining why we are injecting "${getApprovedTabs.toString().slice(9)}" and what it is.